__tests__/app.test.js

```diff
@@ -102,4 +102,52 @@ describe("GET /api/articles", () => {
     });
   });
 
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("responds with an array of comments for the given article_id with the required properties", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          const comments = res.body;
+          comments.forEach((comment) => {
```

Comment:
When using .forEach, we also have to assert against the length of the array. Otherwise, an empty array would automatically pass this test and give us a false positive